chcemvediet/sitemaps.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.sitemaps import Sitemap
+
+from poleno.pages.sitemaps import PagesSitemap
+
+from .apps.inforequests.sitemaps import InforequestSitemap
+
+
+class HomepageSitemap(Sitemap):
+    priority = 1.0
+    changefreq = u'never'
+
+    def items(self):
+        return [u'homepage']
+
+    def location(self, item):
+        return u'/'
+
+sitemaps = {
+        u'homepage': HomepageSitemap,
+        u'pages': PagesSitemap,
+        u'inforequests': InforequestSitemap,
```

Comment:
@viliambalaz Vieme nejak zabespecit poradie prvkov v sitemape? Kedze google niekedy pozrie iba prvych niekolko itemov a dalsie ignoruje, tak prioritnejsie by mali byt na zaciatku. Ci nie je to tak?

Dal by som ich v poradi najprv homepage, potom pages, a az nakoniec inforequesty, ktorych moze byt hodne vela. A tie idealne v poradi, ze novsie skor.

Reply:
@martinmacko47 Docital som sa, ze na poradi prvkov v sitemape nezalezi. V django dokumentacii pre sitemap nie je nic spomenute o zoradovani prvkov. 
Ak by sme chceli zoradovat jednotlive URL, potom by sme asi potrebovali vlastne rozhranie na generovanie sitemap.xml.

Reply:
@viliambalaz Ok, tak na poradie v sitemap kaslime. Vies sem pls pastnut linku, kde si sa to docital?

Reply:
[link 1](https://www.sitemaps.org/faq.html#faq_url_position)
[link 2](https://blog.seoprofiler.com/google-the-order-of-the-urls-in-your-xml-sitemap-does-not-matter/)

Reply:
Ok. Diki. Takze toto neriesme.